synapse/logging/opentracing.py

```diff
@@ -310,6 +310,20 @@ class SynapseTags:
     # The name of the external cache
     CACHE_NAME = "cache.name"
 
+    # Used to tag function arguments
+    #
+    # Tag a named arg. The name of the argument should be appended to this.
+    # prefix
+    FUNC_ARG_PREFIX = "ARG."
+    # Tag extra variadic number of positional arguments (`def foo(first, second, *extras)`)
+    FUNC_ARGS = "args"
+    # Tag keyword args
+    FUNC_KWARGS = "kwargs"
+
+    # Some intermediate result that's interesting to the function. The label for
+    # the result should be appended to this.
+    RESULT_PREFIX = "RESULT."
```

Comment:
Taking inspiration from the `ARG_` tags previously which are really easy to reference because they are in one spot in the alphabetically ordered list, I added a similar `RESULT.` prefix for result type things. Things end up looking like this in Jaeger:

<img width="953" alt="" src="https://user-images.githubusercontent.com/558581/183823817-12b17d90-dcac-44a4-be0c-28ae7c133a3e.png">